backend/dal/Migrations/Initial/PostDeploy/01-ViewProperties.sql

```diff
@@ -0,0 +1,112 @@
+-- This script provides a way to union both parcels and buildings into a single result of properties.
+CREATE VIEW View_Properties AS
```

Comment:
ah, ok. When I saw the configuration I was confused but this makes sense now.

Reply:
I like this view. Would you consider refactoring `PropertiesController.GetProperties()` to use this view or is there anything that would prevent that? Doesn't have to be in the PR, something to think about.

Note that I'm referring to the original `GetProperties` not this new one that is paginated. The original one makes two calls to the database and concatenates the result. Seems like the view would probably offer better performance?